great_expectations/core/expectation_configuration.py

```diff
@@ -1035,36 +1039,30 @@ def patch(self, op: str, path: str, value: Any) -> "ExpectationConfiguration":
         return self
 
     @property
-    def ge_cloud_id(self):
-        if hasattr(self, "_ge_cloud_id"):
-            return self._ge_cloud_id
-        else:
-            return None
+    def ge_cloud_id(self) -> Optional[str]:
+        return self._ge_cloud_id
 
     @ge_cloud_id.setter
-    def ge_cloud_id(self, value):
+    def ge_cloud_id(self, value: str) -> None:
         self._ge_cloud_id = value
 
     @property
-    def expectation_context(self):
-        if hasattr(self, "_expectation_context"):
-            return self._expectation_context
-        else:
-            return None
+    def expectation_context(self) -> ExpectationContext:
+        return self._expectation_context
 
```

Comment:
This was always set in the constructor (even if omitted) so you should always have the attr